From hjk@linutronix.de Sat Feb  2 16:48:40 2008
From: Hans-Jürgen Koch <hjk@linutronix.de>
Date: Sun, 3 Feb 2008 01:48:28 +0100
Subject: UIO: Remove needless PCI_DEVICE_ID definition from uio_cif.c
To: linux-kernel@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Benedikt Spranger <b.spranger@linutronix.de>
Message-ID: <20080203014828.579d6504@dilbert.local>


Meanwhile, PCI_DEVICE_ID_PLX_9030 is defined in pci_ids.h, no need to
define it again in the driver.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
CC: Benedikt Spranger <b.spranger@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/uio/uio_cif.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/uio/uio_cif.c
+++ b/drivers/uio/uio_cif.c
@@ -15,10 +15,6 @@
 
 #include <asm/io.h>
 
-#ifndef PCI_DEVICE_ID_PLX_9030
-#define PCI_DEVICE_ID_PLX_9030	0x9030
-#endif
-
 #define PLX9030_INTCSR		0x4C
 #define INTSCR_INT1_ENABLE	0x01
 #define INTSCR_INT1_STATUS	0x04
